spinalcordtoolbox/scripts/sct_dmri_denoise_patch2self.py

```diff
@@ -45,7 +45,7 @@ def get_parser():
         "-h",
         "--help",
         action="help",
-        help="Show this help message and exit")
+        help="Show this help message and exit.")
```

Comment:
Combining the style pages together sounds good, yes! Especially if it has a table of contents at the top, so that it's possible to skim what are even the points where we express a preference, and also to be able to link to individual points easily.